test/parallel/test-repl-top-level-import.js

```diff
@@ -0,0 +1,145 @@
+'use strict';
```

Comment:
nit, we need to add tests for the following cases so they do not change over time https://github.com/nodejs/node/pull/17285#issuecomment-351454477 .